frontend/src/lib/util.ts

```diff
@@ -183,6 +183,36 @@ export function useErrorState(dependentSetter?: (...args: any) => void) {
   return [error, setError as any];
 }
 
+/**
+ * Combines errors per cluster.
+ *
+ * @param args The list of errors per cluster to join.
+ * @returns The joint list of errors, or null if there are no errors.
+ */
+export function combineClusterListErrors(
```

Comment:
it seems like this function acts very similar to lodash's merge, can it be used instead?

Reply:
I used merge to reduce it a little bit. But because null handling is different it's not a perfect match.